AudioLink/RenderTextures/Materials/rt_AudioLink.mat

```diff
@@ -0,0 +1,81 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
+  serializedVersion: 6
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: rt_AudioLink
+  m_Shader: {fileID: 4800000, guid: 933532a4fcc9baf4fa0491de14d08ed7, type: 3}
```

Comment:
Changes to these .mat files are always a bit opaque. Did you actually change anything meaningful here, if so what?

Reply:
Please revert this change.